raster/r.li/r.li.daemon/daemon.c

```diff
@@ -39,7 +39,7 @@ int calculateIndex(char *file, rli_func *f,
 		   char **parameters, char *raster, char *output)
 {
 
-    char pathSetup[GPATH_MAX], out[GPATH_MAX], parsed;
+    char pathSetup[GPATH_MAX], out[GPATH_MAX + GPATH_MAX], parsed;
```

Comment:
As before, `GPATH_MAX` can not be exceeded, additional tests are needed.

Reply:
see https://github.com/OSGeo/grass/pull/1406#discussion_r595237414